### DISCUSSION TITLE
Kanal 1 eines HMIP-PS kann nicht in einer direkten Verknüpfung als Sender ausgewählt werden

### DISCUSSION THREAD

**Discussion by MarLos68** [OpenCCU/OpenCCU#2486] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Kanal 1 eines HMIP-PS kann nicht in einer direkten Verknüpfung als Sender ausgewählt werden

### Describe the behavior you expected

Ich möchte Kanal 1 (Taster) eines HMIP-PS als Sender für eine neue direkte Verknüpfung benutzen.
Leider kann ich diesen Kanal nicht auswählen, da er nicht angezeigt wird .
In der Auswahl befinden sich unsinnvollerweise nur die Kanäle 3,4 und 5, die aber keine Sender sondern Empfänger sind.

### Steps to reproduce the issue

1.
2.
3.
...

### What is the version this bug report is based on?

3.71.12.20231020

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Nein
```

**Comment by jens-maus** [comment, marked as answer]
Wer sagt denn das man (Nutzer) eine Direktverknüpfung des Tasters an einem HmIP-PS/HmIP-PSM mit einem anderen Gerät vornehmen kann bzw. es möglich sein sollte? Soweit ich sagen kann, kann der Taster lediglich zur lokalen Auslösung des Relais an einem HmIP-PS/HmIP-PSM genutzt werden. Deshalb kann man unter `Direktverknüpfungen` z.B. auch die automatisch angelegte Direktverknüpfung von Kanal :1 und Kanal :3 nicht löschen. Der **Löschen** Knopf fehlt, und es gibt nur ein "Bearbeiten" Knopf, siehe:

<img width="1134" alt="Bildschirmfoto 2023-10-21 um 17 01 08" src="https://github.com/jens-maus/RaspberryMatic/assets/5887769/85410c64-904a-4099-ad13-b75dd5cbf80e">

Insofern wüsste ich nicht einmal ob man diesen Aktor dazu zwingen könnte (wenn man diese Dinge in der WebUI versucht freizuschalten) zwischen :1 und einem anderen Gerät eine Direktverknüpfung zu bauen. Daher sieht das für mich eben danach aus das das kein "Problem" ist, sondern eher halt einfach nicht unterstützt und das es eher `works as designed` ist und daher hier ins Diskussionsforum gehört und nicht zu den issue tickets.

**Comment by Baxxy13** [comment]
Das geht, wenn auch recht kompliziert.
Löschen kann man wenn man zuerst auf bearbeiten geht.
(neu) Anlegen dann mit z.B. Script oder Homematic-Manager.

Zu beachten ist das der Tasterkanal nur eine einzige DV speichern kann. 
[Link Homematic-Forum](https://homematic-forum.de/forum/viewtopic.php?f=65&t=77322&p=754717&hilit=tasterkanal#p754717)

**Reply by jens-maus** [reply]
Danke. Das selbst mir neu. Aber gut zu wissen. D.h. Wir könnten hier zumindest für RaspberryMatic einen WebUI Patch erarbeiten der die Bearbeitung der DV des Kanal :1 zulassen würde? Interessant...

**Reply by Baxxy13** [reply]
Der wird irgendwie geflagged sein und ist somit unsichtbar. 
Wenn man die DV mit dem Taster direkt löschen und auch neu anlegen könnte wäre das nicht schlecht. 
An dem Limit von nur einer DV können wir eher nix ändern, denke das steckt in der Hard/Firmware.
